Makefile

```diff
@@ -12,17 +13,28 @@ all:
 
 sanity:
 	@for file in $$(find lib -type f -name "*.pm" | sort); do \
-		perl -cw -Ilib $$file || exit 1; \
+		$(PERL_BIN)/perl -cw -Ilib $$file || exit 1; \
 	done
 
 test: sanity
 	-$(MAKE) elevate-cpanel
-	/usr/local/cpanel/3rdparty/bin/prove t/00_load.t
-	/usr/local/cpanel/3rdparty/bin/yath test -j8 t/*.t
+	$(PERL_BIN)/prove t/00_load.t
+	$(PERL_BIN)/yath test -j8 t/*.t
+
+build:
```

Comment:
could we have a comment to explain what this target is used for?

Reply:
This target is used to fatpack the script.  I use this very frequently on my sandbox.  Please add another target to perform this task instead of altering the `build` target.

Reply:
Moved into a target **fatpack**